tests/telemetry/test_telemetry.py

```diff
@@ -243,19 +242,37 @@ def test_app_started_error_unhandled_exception(test_agent_session, run_python_co
     assert "Unable to parse DD_SPAN_SAMPLING_RULES='invalid_rules'" in app_starteds[0]["payload"]["error"]["message"]
 
 
-def test_telemetry_with_raised_exception(test_agent_session, run_python_code_in_subprocess):
-    _, stderr, status, _ = run_python_code_in_subprocess(
-        "import ddtrace; ddtrace.tracer.trace('moon').finish(); raise Exception('bad_code')"
+def test_register_telemetry_excepthook_after_another_hook(test_agent_session, run_python_code_in_subprocess):
+    out, stderr, status, _ = run_python_code_in_subprocess(
+        """
+import sys
+
+old_exc_hook = sys.excepthook
+def pre_ddtrace_exc_hook(exctype, value, traceback):
+    print("pre_ddtrace_exc_hook called")
+    return old_exc_hook(exctype, value, traceback)
+
+sys.excepthook = pre_ddtrace_exc_hook
+
+import ddtrace
+raise Exception('bad_code')
+"""
     )
+    assert b"pre_ddtrace_exc_hook called" in out
     assert status == 1, stderr
     assert b"bad_code" in stderr
     # Regression test for python3.12 support
     assert b"RuntimeError: can't create new thread at interpreter shutdown" not in stderr
+    # Regression test for invalid number of arguments in wrapped exception hook
+    assert b"TypeError: pre_ddtrace_exc_hook() takes 3 positional arguments but 4 were given" not in stderr
```

Comment:
```suggestion
    assert b"3 positional arguments but 4 were given" not in stderr
```

non-blocking nit, just makes it a bit more robust in case the function name or something else changes.